website/docs/r/storage_account.html.markdown

```diff
@@ -344,23 +344,21 @@ A `azure_files_authentication` block supports the following:
 
 * `active_directory` - (Optional) A `active_directory` block as defined below. Required when `directory_type` is `AD`.
```

Comment:
Can we add a note above for `directory_type` clarifying that `domain_sid`, `storage_sid`, `forest_name`, `netbios_domain_name` are required when `directory_type` is set to `AD`?

Reply:
They are mentioned for each of the property themselves, do you mean we need to add an additional note something like:

```
~> **NOTE:** `domain_sid`, `storage_sid`, `forest_name`, `netbios_domain_name` are required when `directory_type` is set to `AD`.

A `active_directory` block supports the following:

* `domain_name` - (Required) Specifies the primary domain that the AD DNS server is authoritative for.

* `domain_guid` - (Required) Specifies the domain GUID.

...

```

Reply:
Ah apologies, I overlooked that sentence in the property description. In which case that's fine, nothing to do here.